replication-api/src/main/java/com/connexta/replication/queue/TaskInfo.java

```diff
@@ -20,11 +20,11 @@
 /** Represents a task that can be queued for later processing by a worker. */
 public interface TaskInfo {
   /**
-   * Gets the priority for this task.
+   * Gets the priority for this task, represented as a number between 1 and 10.
    *
    * @return the priority for the task
    */
-  public PriorityLevel getPriority();
```

Comment:
❗️ Remove the PriorityLevel enum

Reply:
I did apparently git thinks I changed it to be the filterManager interface.